src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -170,19 +164,17 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
         }
 
         /// <summary>
-        /// get payload as an asynchronous operation.
+        /// Gets an HTTP payload as as an asynchronous operation.
```

Comment:
_"Gets an HTTP payload as an asynchronous operation."_ (_as_ was duplicated)

Reply:
Sorry, noticed that was my mistake in the original suggestion, but it was still wrong 😄